cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -735,11 +735,22 @@ def _create_views_with_node_types(
                             "Multiedge relation must have a view as value type. "
                             "This should have been validated in the rules"
                         )
-                    view_property = dm.MultiEdgeConnectionApply(
-                        type=dm.DirectRelationReference(
+                    if isinstance(prop.reference, ReferenceEntity):
+                        ref_view_prop = prop.reference.as_prop_id(
+                            default_space, default_version, self.standardize_casing
```

Comment:
I understood from the previous PR that `standardized_casing` was causing issue and that you removed it?